### PR TITLE
Detectors/ITSMFT: make readability-braces-around-statements happy

### PR DESCRIPTION
**Pull request by anerokhi**

### CODE REVIEW

**Review by bovulpes — APPROVED** [review]

**Comment by mpuccio** [comment]
@sawenzel @anerokhi guys, this makes our life difficult: if you have such modifications to the code at least wait for the code authors approval. At least one of the approvers was not aware/did not sign off. We have an open PR where we now need to do a manual rebase.

**Comment by sawenzel** [comment]
I apologize but this a difficult call to make. It is better to finish this fixing quickly so that we can enable the actual check.
Sometimes it is hard to judge whether an approval will come or not come and these are in any case trivial fixes which do not break any functionality. Let me know if I can help with the merge conflict.

**Pull request merged**
